platform/linux-generic/odp_classification.c

```diff
@@ -89,8 +102,22 @@ int odp_classification_init_global(void)
 		LOCK_INIT(&pmr->s.lock);
 	}
 
+	queue_grp_shm = odp_shm_reserve("shm_odp_cls_queue_grp_tbl",
+					sizeof(_cls_queue_grp_tbl_t),
+					sizeof(queue_entry_t *), 0);
+
```

Comment:
This area is allocated but never freed. It should be in odp_classification_term_global(). Running the classification validation test gives the error:
```
_ishm.c:1659:_odp_ishm_term_global():block 'shm_odp_cls_queue_grp_tbl' (file ) was never freed (cleaning up...).
```